### ISSUE TITLE
Bumping grocy version to 2.4.0

### ISSUE THREAD

**Issue by kingo55** [grocy/grocy-docker#16]
I pulled the grocy-docker repo here to try upgrading it but the image build won't work (likely my version of openssl on my main rig). Just throws this during the build - haven't got time to fix today either:

```
SHA384 is not supported by your openssl extension, could not verify the phar file integrity
```

Anyone tried building for 2.4.0 or even 2.3.0 yet?

**Comment by kilakewe** [comment]
This is due to the version of composer being used. It looks like the script is pulling a specific version.

**Comment by kilakewe** [comment]
After updating it, there's a new issue to resolve:

Warning: include(/www/data/viewcache/4b3e495a8ce1359c2f68eb6c33e07337aa6b2a0f.php): failed to open stream: No such file or directory in /www/vendor/illuminate/view/Engines/PhpEngine.php on line 43

Warning: include(): Failed opening '/www/data/viewcache/4b3e495a8ce1359c2f68eb6c33e07337aa6b2a0f.php' for inclusion (include_path='.:/usr/local/lib/php') in /www/vendor/illuminate/view/Engines/PhpEngine.php on line 43

Warning: file_put_contents(/www/data/viewcache/4b3e495a8ce1359c2f68eb6c33e07337aa6b2a0f.php): failed to open stream: No such file or directory in /www/vendor/illuminate/filesystem/Filesystem.php on line 122

**Comment by kilakewe** [comment]
A pull request has been submitted to resolve this.

**Comment by schildgenb** [comment]
Has this been resolved yet? Is there anything that can be done manually?

**Comment by r1co** [comment]
@berrnd Can you please merge @kilakewe's pullrequest? 
I would love to have to mealplan feature :)

**Comment by berrnd** [comment]
grocy-docker is not my project, @talmai created and cared about it so far (I personally still not using Docker for anything I host)...

**Comment by talmai** [comment]
APOLOGIES to all. Apparently, only when I am explicitly referenced (@) do I get email alerts. Will look into this, this weekend.

**Comment by berrnd** [comment]
Thank you! :)

Just off topic (@talmai):
If you "watch" the repo you should get mails about all issue comments, if you want this.

![grafik](https://user-images.githubusercontent.com/2161815/59788303-b12c5600-92cb-11e9-9122-ee069561bc48.png)

**Comment by talmai** [comment]
merged + updated to 2.4.4
